### ISSUE TITLE
noise fx issue

### ISSUE THREAD

**Issue by reset-selectgame-1-9** [Grabulosaure/Intv_MiSTer#1]
Hi,
Firstly I have to say that it's amazing that somebody is working on an Intellivision core for MiSTer. I tried already your latest version with using my Flashback Controllers via Ultimate PC Interface (https://bit.ly/2lPHGhI). Disc and all the buttons are working properly!
I think soundwise there seems to be a small issue with the noise FX. It sounds to high and aggressive. I played Skiing and compared it on my original console. Here's a video to compare: https://youtu.be/hdUek9I0F9M
The other waveform (Wave or Pluse) seems to sound spot on.
Many thanks for all your work in making this core!

**Comment by Grabulosaure** [comment]
Yes. I've used for the sound chip some code I wrote a while ago for Vectrex, without much testing. I'll check the other implementations of similar chips (AY-3-891x)

**Comment by reset-selectgame-1-9** [comment]
Noise is much more authentic with the latest release (2019-09-21), thanks!

**Comment by ugrash2097** [comment]
Since the latest release (2021-03-18), there's some new problem. In Auto Racing, the noise at start (tires I think) never stops and continue during all the race. In Astrosmash, the 'tac, tac' is weird, like there's some reverb.
